### ISSUE TITLE
webview_flutter plugin is not accessible (VoiceOver)

### ISSUE THREAD

**Issue by diegovar** [flutter/flutter#30655]
The webview_flutter plugin does behave correctly with VoiceOver in iOS. Tapping the webview while having VoiceOver enabled has no effect, and nothing can be triggered in the webview in this way.

**Comment by escamoteur** [comment]
Hi, sorry for the delay, I just want to check if you could solve this problem.

**Comment by iapicca** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!
  
Thanks for your contribution.

**Comment by armandojimenez** [comment]
Any updates to this? Voice over does not work on WebViews, any section is not clickable

**Comment by iapicca** [comment]
Could everyone who still has this problem please file a new issue with the exact description of what happens, logs and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
